src/data-access/repositories/mongo/mongo-database.ts

```diff
@@ -3,8 +3,10 @@ import { Collection } from 'mongodb'
 import { DatabaseNotConnectedException } from '../../../model/exceptions/database-not-connected-exception'
 
 // TODO: Move to ENV variables
-const MONGO_CONNECTION_STRING: string = 'mongodb://localhost:3001'
-const MONGO_DB_NAME: string = 'meteor'
+// const MONGO_CONNECTION_STRING: string = 'mongodb://localhost:3001'
+// const MONGO_DB_NAME: string = 'meteor'
+const MONGO_CONNECTION_STRING: string = 'mongodb://localhost:27017?directConnection=true'
+const MONGO_DB_NAME: string = 'sofie'
```

Comment:
This was accidentally committed. It's removed in my PR for persisting audio for cameras.

Reply:
Sidenote: This is how you connect to the MongoDB for Staging

Reply:
If it isn't already noted down somewhere in e.g confluence, then it might be worth doing that at some point.

Reply:
I plan on writing a confluence document(s) about the how the all this works once I'm done with building the timeline.
This could be a thing to include for our local developer guide.